### PR TITLE
reformat with scalafmt 2.5.2

### PR DESCRIPTION
**Pull request by pshemass**

### CODE REVIEW

File: core/src/main/scala/zio/logging/LogLevel.scala [hunk 1]
```diff
@@ -16,7 +16,8 @@ sealed trait LogLevel { self =>
 
   def min(that: LogLevel): LogLevel = if (self > that) that else self
 }
-object LogLevel {
+object LogLevel       {
```
Comment (pshemass) [inline core/src/main/scala/zio/logging/LogLevel.scala:19]: @mijicd do you know how to get rid of this?

Comment (mijicd) [reply core/src/main/scala/zio/logging/LogLevel.scala:19]: Yes, you need to wrap it like this:

```scala
// format: off
object Dummy {
// format: on
```

Comment (pshemass) [reply core/src/main/scala/zio/logging/LogLevel.scala:19]: this is disgusting :(

Comment (mijicd) [reply core/src/main/scala/zio/logging/LogLevel.scala:19]: I'm not sure what's uglier, the formatting or these "magic" comments.

Comment (pshemass) [reply core/src/main/scala/zio/logging/LogLevel.scala:19]: I agree it's pretty hard to judge :D

**Pull request merged**